docs/src/main/sphinx/release/release-464.md

```diff
@@ -15,6 +15,11 @@
 
 * {{breaking}} Remove the Accumulo connector. ({issue}`23792`)  
 
+## BigQuery connector
+
+* Fix array data type column returning incorrect results when
```

Comment:
Maybe "Fix incorrect results when reading array columns and bigquery.arrow-serialization.enabled is set to true."